modules/aws_autoscaling/main.tf

```diff
@@ -21,7 +25,9 @@ data "aws_iam_policy_document" "worker_autoscaling" {
     actions = [
       "autoscaling:SetDesiredCapacity",
       "autoscaling:TerminateInstanceInAutoScalingGroup",
-      "autoscaling:UpdateAutoScalingGroup",
```

Comment:
Just confirming this action removal is expected as you mentioned the policy update was additive.

Reply:
Looks like I copy pasted wrong. That action is only needed for the older version of the autoscaler (9.9.2), going to add that back and run a quick test. I'll reply here once verified.

Reply:
@dhoucgitter ran through both the 9.9.2 and 9.25.0 scenario after I made the updates. Both scenarios ran fine and nodes scaled up and down. Ready for re-review.